akka-docs/src/main/paradox/project/migration-guide-2.4.x-2.5.x.md

```diff
@@ -690,7 +690,7 @@ implicit val materializer = ActorMaterializer.create(system)
 
 val throttler: ActorRef =
   Source.actorRef(bufferSize = 1000, OverflowStrategy.dropNew)
-    .throttle(100, 1.second, 10, ThrottleMode.Shaping)
+    .throttle(100, 1.second)
```

Comment:
Words cannot describe how happy I am with this new signature. 🥇

Reply:
Yeah same here 👍